### PR TITLE
Add Style

### PR DESCRIPTION
**Pull request by chelobotix**
In this PR I fulfilled the following task:
- [x] Add a Style that matches Zeplin's design
- [x] Add missing JSX elements to the project

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,9 +1,8 @@
 import { BrowserRouter, Routes, Route } from 'react-router-dom';
-import './App.css';
 import { useEffect } from 'react';
 import { useSelector, useDispatch } from 'react-redux';
 import { fetchGetBooks } from './redux/books/booksSlice';
-
+import './app.css';
```
Comment (OLIPLICHE) [inline src/App.js]: - Hi, There is a typo mistake you could fix here. I would suggest you to change `app.css` to `App.css`
Capture

File: src/components/AddBook.js [hunk 2]
```diff
@@ -2,6 +2,7 @@
 import { useSelector, useDispatch } from 'react-redux';
 import { fetchGetBooks, fetchPostBook } from '../redux/books/booksSlice';
 import { v4 as uuidv4 } from 'uuid';
```
Comment (OLIPLICHE) [inline src/components/AddBook.js]: - Hi, I would advise you not to disable linters, you can solve the bugs manually. 
- ESLint is a tool for identifying and reporting on patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.
Capture

File: src/components/Book.js [hunk 3]
```diff
@@ -1,9 +1,13 @@
...
+import loading from '../assets/images/loading.png';
 
 const Book = (props) => {
-  const { id, author, title } = props;
+  const {
+    id, author, title, category,
+  } = props;
   const dispatch = useDispatch();
 
   const handleDeleteBook = (id) => {
```
Comment (OLIPLICHE) [inline src/components/Book.js:13]: - [x] Hi, I would advise you not to disable linters, you can solve the bugs manually.

- [x] ESLint is a tool for identifying and reporting patterns found in ECMAScript/JavaScript code, with the goal of making code more consistent and avoiding bugs.


Capture

**Review by OLIPLICHE — CHANGES_REQUESTED** [review]
Hi @chelobotix , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- README file is descriptive 👏
- Commit messages are meaningful 👏 👏
- Gitflow is correctly used (Pull Request is made from the feature branch into the dev branch) 👏

## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by chelobotix** [comment]
All Observations have been implemented.

**Review by pinkmoon25 — APPROVED** [review]
Hi @chelobotix , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



## Highlights
linter checks :heavy_check_mark: 
gitflow :heavy_check_mark: 
Styling :heavy_check_mark: 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**